### DISCUSSION TITLE
Add text to footer

### DISCUSSION THREAD

**Discussion by snaqvi1990** [greenelab/lab-website-template#236] [Q&A] [answered]
HI again! Is there a way to add text (i.e. for copyrighted images) to the footer text that says "Built with Lab Website Template" etc?

**Comment by vincerubinetti** [comment, marked as answer]
Yes, see `/_includes/footer.html` and edit as desired. You can replace the copyright text, or add another div row there to add a new line for more text.

**Comment by snaqvi1990** [comment]
Great, thanks!
